include/links-directory.php

```diff
@@ -81,12 +81,9 @@ public function add_language_to_link( $url, $language ) {
 	 * @return string The modified url.
 	 */
 	public function remove_language_from_link( $url ) {
-		$languages = $this->model->get_languages_list(
-			array(
-				'hide_default' => $this->options['hide_default'],
-				'fields'       => 'slug',
-			)
-		);
+		$languages = $this->model->languages
+			->filter( $this->options['hide_default'] ? 'hide_default' : '' )
+			->get_list( array( 'fields' => 'slug' ) );
```

Comment:
`array( 'fields' => 'slug' )` is the only remaining param for `get_list()`. What do you think to progressively remove its usage and inline the call to `wp_list_pluck()` when we need it?